packages/vite/src/node/server/middlewares/proxy.ts

```diff
@@ -141,4 +145,4 @@ function doesProxyContextMatchUrl(context: string, url: string): boolean {
     (context.startsWith('^') && new RegExp(context).test(url)) ||
     url.startsWith(context)
   )
-}
+}
```

Comment:
Would you run `pnpm format`?

Reply:
> Would you run `pnpm format`?

sorry＞︿＜

Reply:
> Would you run `pnpm format`?

@sapphi-red 
Could you format it for me？There is a problem with our network

Reply:
No problem. OK 😃